src/rgw/radosgw-admin/radosgw-admin.cc

```diff
@@ -395,6 +399,7 @@ void usage()
   cout << "   --url=<url>                       url for pushing/pulling period/realm\n";
   cout << "   --epoch=<number>                  period epoch\n";
   cout << "   --commit                          commit the period during 'period update'\n";
+  cout << "   --dry-run                         dry run only, no changes will be made\n";
```

Comment:
This operation only collects stats, and we say above that no changes are made, so why do we need this flag?

Reply:
Without the --dry-run flag we are doing the full dedup